runtime/service_integ_test.go

```diff
@@ -2469,6 +2469,8 @@ func TestPauseResume_Isolated(t *testing.T) {
 		_, err := fcClient.ResumeVM(ctx, &proto.ResumeVMRequest{VMID: vmID})
 		require.NoError(t, err)
 
+		time.Sleep(3 * time.Second)
```

Comment:
leftover artifact?

Reply:
Makes sense, for commit history reasons this feels weird to include in this PR since we do squash and merge but I guess if it's just one line it's not that bad 🤷